docs/hardware/srobo/sbv4.rst

```diff
@@ -1,7 +1,134 @@
+| LED           | Meaning                                                                                          | Initial power-up state |
++===============+==================================================================================================+========================+
+| Power         | The board is powered over USB.                                                                   | On                     |
++---------------+--------------------------------------------------------------------------------------------------+------------------------+
+| 5.5V On       | There is 5.5V power on the board. This usually indicates that the 12V connector rail is powered. | Off                    |
++---------------+--------------------------------------------------------------------------------------------------+------------------------+
+| Aux On        | There is auxiliary power on the board.                                                           | Off                    |
++---------------+--------------------------------------------------------------------------------------------------+------------------------+
+| Status        | The board has successfully booted.                                                               | On                     |
++---------------+--------------------------------------------------------------------------------------------------+------------------------+
+| Error         | An error has occurred. This often indicates a disconnection of the 12V connector.                | Off                    |
++---------------+--------------------------------------------------------------------------------------------------+------------------------+
+| Outputs 00-11 | Currently unimplemented in firmware.                                                             | Off                    |
++---------------+--------------------------------------------------------------------------------------------------+------------------------+
+
+Controls
+--------
+
+The servo board is largely controlled over the USB interface. There is one physical push button on the board that is a firmware reset button.
+
+USB Interface
+-------------
+
+The Vendor ID is ``1bda`` (University of Southampton) and the product ID is ``0011``.
+
+The Servo Board is controlled over USB by sending requests to the control endpoint.
+
+.. code-block:: python
+
+    ctrl_transfer(
+        0x00,
+        64,
+        wValue=req_val,
+        wIndex=command.code,
+        data_or_wLength=req_data,
+    )
+
++---------------+-------+
+| Parameter     | Value |
++===============+=======+
+| bmRequestType | 0x00  |
++---------------+-------+
+| bRequest      | 64    |
++---------------+-------+
+
+There are a list of ids defined in the firmware of the servo board that will let you read and write values to it.
+
+It is recommended to read the source to further understand how to control this device.
```

Comment:
```suggestion
It is recommended to read the `firmware source`_ to further understand how to control this device.
```

I think this should use the same hyperlink target as the link at the bottom of the page, but I'm not familiar enough with reStructuredText to be sure.

Reply:
Yeah it should